website/config/framedata/fwireframe.json

```diff
@@ -1 +1,222 @@
,"normalizedName":"uair","hits":[],"landLag":30,"lCanceledLandLag":15,"totalFrames":0,"start":16,"end":17,"type":3,"characterId":0,"source":"https://fightcore.gg/","gifUrl":"https://i.fightcore.gg/melee/moves/fwireframe/uair.gif","webmUrl":"https://i.fightcore.gg/melee/moves/fwireframe/uair.webm","isInterpolated":false,"sources":[],"alternativeAnimations":[],"id":1961},{"name":"Forward Air","normalizedName":"fair","hits":[],"landLag":30,"lCanceledLandLag":15,"totalFrames":0,"start":8,"end":11,"type":3,"characterId":0,"source":"https://fightcore.gg/","gifUrl":"https://i.fightcore.gg/melee/moves/fwireframe/fair.gif","webmUrl":"https://i.fightcore.gg/melee/moves/fwireframe/fair.webm","isInterpolated":false,"sources":[],"alternativeAnimations":[],"id":1962},{"name":"Down Air","normalizedName":"dair","hits":[],"landLag":30,"lCanceledLandLag":15,"totalFrames":0,"start":14,"end":17,"type":3,"characterId":0,"source":"https://fightcore.gg/","gifUrl":"https://i.fightcore.gg/melee/moves/fwireframe/dair.gif","webmUrl":"https://i.fightcore.gg/melee/moves/fwireframe/dair.webm","isInterpolated":false,"sources":[],"alternativeAnimations":[],"id":1963},{"name":"Neutral Air","normalizedName":"nair","hits":[],"landLag":30,"lCanceledLandLag":15,"totalFrames":0,"start":6,"end":27,"type":3,"characterId":0,"source":"https://fightcore.gg/","gifUrl":"https://i.fightcore.gg/melee/moves/fwireframe/nair.gif","webmUrl":"https://i.fightcore.gg/melee/moves/fwireframe/nair.webm","isInterpolated":false,"sources":[],"alternativeAnimations":[],"id":1964},{"name":"Up Smash","normalizedName":"usmash","hits":[],"totalFrames":56,"iasa":49,"start":10,"end":29,"type":2,"characterId":0,"source":"https://fightcore.gg/","gifUrl":"https://i.fightcore.gg/melee/moves/fwireframe/usmash.gif","webmUrl":"https://i.fightcore.gg/melee/moves/fwireframe/usmash.webm","isInterpolated":false,"sources":[],"alternativeAnimations":[],"id":1965},{"name":"Forward Smash","normalizedName":"fsmash","hits":[],"totalFrames":40,"iasa":40,"start":16,"end":16,"type":2,"characterId":0,"source":"https://fightcore.gg/","gifUrl":"https://i.fightcore.gg/melee/moves/fwireframe/fsmash.gif","webmUrl":"https://i.fightcore.gg/melee/moves/fwireframe/fsmash.webm","isInterpolated":false,"sources":[],"alternativeAnimations":[],"id":1966},{"name":"Down Smash","normalizedName":"dsmash","hits":[],"totalFrames":40,"iasa":34,"start":4,"end":16,"type":2,"characterId":0,"source":"https://fightcore.gg/","gifUrl":"https://i.fightcore.gg/melee/moves/fwireframe/dsmash.gif","webmUrl":"https://i.fightcore.gg/melee/moves/fwireframe/dsmash.webm","isInterpolated":false,"sources":[],"alternativeAnimations":[],"id":1967}],"characterStatistics":{"id":53,"weight":0,"gravity":0.0,"walkSpeed":0.0,"runSpeed":0.0,"waveDashLengthRank":0,"plaIntangibilityFrames":0,"jumpSquat":0,"canWallJump":false,"notes":"No data available yet","initialDash":0.0,"dashFrames":0,"waveDashLength":0.0},"characterInfo":{"id":27,"discord":"https://discord.gg/VT3H5PN"},"id":53}
```

Comment:
I'd prefer if we keep the JSON files here one line. These are generated and not manually edited.

Reply:
With "here" I mean all in the `config/framedata` folder.

Reply:
Added all JSON files in the `config/framedata` folder to the `.prettierignore` and reverted the changes on the JSON files.